index.html

```diff
@@ -135,9 +128,10 @@ <h3>Youtube Video</h3>
         </p>
 
         <picture>
-          <source media="(min-width: 70px)" srcset="Images/sc-logo@1.5x.png 150px" />
-          <source media="(min-width: 70px)" srcset="Images/sc-logo@2.5x.png 175px"/>
-          <img src="Images/sc-logo.png" alt=" SC Logo" />
+          <source type="image/svg+xml" srcset="Images/sc-logo.svg" />
+          <img src="Images/sc-logo.png" width="70"
+            srcset="Images/sc-logo.png, Images/sc-logo@1.5x.png 1.5x, Images/sc-logo@2x.png 2x"
+            alt=" SC Logo" />
```

Comment:
Close.  In this case, use `<source>` to supply the SVG, and srcset to supply the alternate resolutions